rails-tasks/app/controllers/tasks_controller.rb

```diff
@@ -8,10 +8,8 @@ def index
 
   def show
     @task_name = params[:name]
-    task = Task.where(name: @task_name)[0]
-    @description = task.description
-    @completed_status = task.complete
-    date = task.completed_date
+    @task = Task.find_by(name: params[:name])
+    date = @task.completed_date
```

Comment:
Rather than check the date variable in the controller we should pass it directly to the template (with `@completed_date = @task.completed_date`) and have the view logic, ideally through a helper method, actually determine the formatting (which we are currently doing on lines 16 and 43).